Mail/Views/Unavailable Mailbox/UpdateMailboxPasswordView.swift

```diff
@@ -30,6 +30,7 @@ struct UpdateMailboxPasswordView: View {
     @State private var updatedMailboxPassword = ""
     @State private var isShowingError = false
     @State private var isLoading = false
+    @State private var showDetachMailboxAlert = false
```

Comment:
Could we start enforcing using `is` at the beginning of boolean properties that show views ?
And always include View in the property.
-> isShowDetachMailboxAlertView